client/extensions/woocommerce/state/wc-api/products/actions.js

```diff
@@ -2,13 +2,23 @@
  * Internal dependencies
  */
 import wp from 'lib/wp';
+
 import { error } from '../actions';
 import {
 	WOOCOMMERCE_API_CREATE_PRODUCT,
 	WOOCOMMERCE_API_CREATE_PRODUCT_SUCCESS,
 } from '../../action-types';
 
-export function createProduct( siteId, product ) {
+/**
+ * API call to create a product within designated WooCommerce site.
+ *
+ * @param {number} siteId The Jetpack Site ID for the WooCommerce WordPress site.
```

Comment:
Should this be `Number` ? Also, usually we just comment this param as `Site ID` even for Jetpack sites.